book/src/versions/0.15/2023-04-27.md

```diff
@@ -1,26 +1,44 @@
-# 2023-03-28 Updates
+# 2023-04-27 Updates
 
 We've published minor versions of a number of crates. This release is composed
-of bugfixes, performance improvements, and documentation improvements.
+of bugfixes, new features, performance improvements, and documentation
+improvements.
 
 ## Bugfixes
 
+Discord's Clyde AI bot has a unique avatar hash. Unlike other hashes that are
+hex based, Clyde's avatar hash is simply "clyde". We now handle deserialization
+of Clyde's avatar in our [`ImageHash`] optimization.
+
+Application command interaction options of type `String` are no longer trimmed
+of leading zeroes.
+
+The [`UpdateCurrentMember`] request now correctly nulls the current member's
```

Comment:
"nulls" is JSON-speak, "clears" or "removes" would be the actual effect of the request.